guides/micronaut-jms-aws-sqs/kotlin/src/test/kotlin/example/micronaut/MicronautguideTest.kt

```diff
@@ -0,0 +1,46 @@
+package example.micronaut
+
+import io.micronaut.core.annotation.NonNull
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import io.micronaut.test.support.TestPropertyProvider
+import jakarta.inject.Inject
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.testcontainers.junit.jupiter.Testcontainers
+
+@Testcontainers(disabledWithoutDocker = true)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@MicronautTest
+internal class MicronautguideTest : TestPropertyProvider {
+
+    @Inject
+    lateinit var demoController: DemoController
```

Comment:
I would prefer if we injected a HTTP Client and invoke the controller from the outside. 

Replace 

```
@Inject
DemoController demoController
```

with 

```
@Inject
@Client("/")
HttpClient httpClient
````

replace: 

`demoController.publishDemoMessages()`

with: 

`httpClient.toBlocking().exchange(HttpRequest.POST("/demo", Collections.emptyMap())`